### ISSUE TITLE
Page tools less usable for low vision users

### ISSUE THREAD

**Issue by selfthinker** [splitbrain/dokuwiki#3452]
* [ ] When resizing text in Firefox with "zoom text only" and hovering or focusing on the page tools, the text size increases but the highlighted area and the overall container doesn't. That makes it harder to understand which item is highlighted. And the bottom item overlaps with the bottom border and page content, making it harder to read.
That fails [1.4.4 Resize text](https://www.w3.org/TR/WCAG21/#resize-text).

![](https://user-images.githubusercontent.com/108893/112806134-60d05380-906e-11eb-85c5-cb8376739d44.png)

## Other non-fail accessibility issues

These don't strictly fail WCAG but are still accessibility issues. Fixing them is recommended but optional for passing WCAG.

* [ ] When resizing text in Firefox with "zoom text only", the page tool icons don't get bigger. That makes them harder to see for low vision users, especially combined with their low colour contrast (as listed in #3443).

* [ ] When using a screen magnifier, the page tools are much harder to notice and find on the page. As soon as you know where they are, that will be less of a problem, though.

* [ ] When I tested activating items in the page tools via voice control (Dragon), I was able to click on each item (via saying e.g. "click backlinks") but only because I knew what to say. You cannot hover over anything with voice control. You could tab to it, but voice control users rarely do that. Instead they would just click. And for that they would need to know what the thing is called that they want to click on. But if they've never seen the page tools open, they will never see their names. That means there is more cognitive load and uncertainty for people with motor impairments. They would probably rely on 'mouse grid' instead (which lets you zone in on things to click on) which takes longer to use than just saying what to click on.
